src/info.js

```diff
@@ -37,7 +37,7 @@ const getFunctionInfo = async function (srcPath) {
     return {}
   }
 
-  if (extension === '.zip' || extension === '.js') {
+  if (['.js', '.ts', '.zip'].includes(extension)) {
```

Comment:
Should we add `.cjs` and `.mjs`?
There is also the older / less common `.es6`.

Reply:
To revisit once we add support for additional extensions.